app/controllers/hiring_staff/sign_in/dfe/sessions_controller.rb

```diff
@@ -21,4 +22,8 @@ def create
   private def oid
     auth_hash['uid']
   end
+
+  private def identifier
+    auth_hash['uid']
```

Comment:
this should be updated to retrieve the email, unless we opt for using different mappings for each service DFE Sign in vs Azure.

Reply:
@despo  This PR returns the information from DfE Sign In and includes an example payload: https://github.com/dxw/teacher-vacancy-service/pull/344